finesse/gui/led_icons.py

```diff
@@ -37,8 +37,8 @@ def __init__(self, on_img: QImage, off_img: QImage, is_on: bool = False) -> None
             self._turn_on()
         else:
             self._turn_off()
-        self._timer = QTimer()
-        self._timer.timeout.connect(self._turn_off)  # type: ignore
+        self.timer = QTimer()
+        self.timer.timeout.connect(self._turn_off)  # type: ignore
```

Comment:
Why is `timer` public?

Reply:
Because it is used outside of the class. See my comments above.

Reply:
Ohhh I see.